peru/resources/plugins/curl/curl_plugin.py

```diff
@@ -9,10 +9,26 @@
 import tarfile
 from urllib.error import HTTPError, URLError
 from urllib.parse import urlsplit
+from urllib.request import Request
+import peru.main
 import urllib.request
 import zipfile
 
 
+def peru_version():
+    return peru.main.get_version()
```

Comment:
This could probably be inlined. I stubbed it to make vim-flake8 shut up and then didn't inline it myself.